app/src/main/java/org/mozilla/focus/home/HomeFragment.java

```diff
@@ -162,9 +158,16 @@ public void onStop() {
         }
     }
 
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        doWithActivity(getActivity(), themeManager -> themeManager.subscribeThemeChange(homeScreenBackground));
```

Comment:
I tend to think that onStart() / onStop() are a better place for these. Or maybe onCreateView() / onDestroyView().

Reply:
I think onActivityResult is the closet hook after onCreateView, and the subscription needs to happen before Fragment becomes visible to make sure all notify will be handled. It could possibly better if we do this in onCreateView, but I tend to make onCreateView to only View creation things.